src/Adapter/Session/Repository/CustomerSessionRepository.php

```diff
@@ -141,7 +141,7 @@ public function clearOutdatedSessions(): void
 
             $qb = $this->connection->createQueryBuilder();
             $qb->delete($this->dbPrefix . 'customer_session')
-                ->where('date_upd < :dateUpdated')
+                ->where('date_upd <= :dateUpdated')
```

Comment:
Is this related to this PR?

Reply:
No but as I explained on slack the behat tests were failing I already encountered this bug a few weeks ago, I have the feeling that it happened more at night when there more resources available for CI and they can run faster
My guess is that the test used to pass by chance because enough time had passed to interprate the expiration, bit if it runs fast the values were equal, the condition used AND the data in the scenario were too close to be correctly interprated